### PR TITLE
fix typo (utilize)

### PR DESCRIPTION
**Pull request by OmarHussari**
### Description

Fixed a typo (the word "utilize") for a project in the Web Dev Skillpath at Codecademy.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/ajax/ajax.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ CatalogContent:
 
 ## Syntax
 
-AJAX requests can be used in all modern web browsers and utliizes the built-in `XMLHttpRequest` object:
+AJAX requests can be used in all modern web browsers and utilizes the built-in `XMLHttpRequest` object:
```
Comment (KTom101) [inline content/javascript/concepts/ajax/ajax.md:23]:
```suggestion
AJAX requests can be used in all modern web browsers and it utilizes the built-in `XMLHttpRequest` object:
```

**Review by KTom101 — COMMENTED** [review]
@OmarHussari Thanks for finding this. I've added a minor change.

File: content/javascript/concepts/ajax/ajax.md (hunk 1 above)
Comment (OmarHussari) [reply content/javascript/concepts/ajax/ajax.md:23]: You're very welcome :D

**Review by KTom101 — APPROVED** [review]

**Comment by KTom101** [comment]
@OmarHussari The live changes are here: https://www.codecademy.com/resources/docs/javascript/ajax

I noticed however you are not listed as a contributor. If you wish to be added please make sure you have your GitHub username listed under your Codecademy profile by following these steps:

1. Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

Screen Shot 2023-01-08 at 7 56 27 PM

2. Then log out and log back into Codecademy using your GitHub login:

Log in to Codecademy

Let me know if you need any help if you decide to do so.

**Pull request merged**